### DISCUSSION TITLE
zeebe-dmn

### DISCUSSION THREAD

**Discussion by Milancho** [camunda-community-hub/community#115] [Q&A]
I have a use case where I need to evaluate DMN decisions (i.e. business rule tasks) without including them in the BPMN diagram.
Any suggestions on how I can achieve this using C# and Zeebe worker on-premise?
I need to upload DMN and get results.

Thank you for all the help!

**Comment by xomiamoore** [comment]
Hi @Milancho, please post your question in the [forums](https://forum.camunda.io/). That's the best way to get an answer as this GitHub community is for our community projects, plugins, and other tools.
